Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Ensure that the `COPY . .` command is used correctly. It should copy the contents of the current directory to the `WORKDIR` set earlier. Verify that the `app/main.py` file exists in the source directory to avoid runtime errors.